src/application.rs

```diff
@@ -1,15 +1,15 @@
-use Application;
-use File;
-use ffi;
-use glib::GString;
+use gio_sys;
 use glib::object::Cast;
 use glib::object::IsA;
-use glib::signal::{SignalHandlerId, connect_raw};
+use glib::signal::{connect_raw, SignalHandlerId};
 use glib::translate::*;
-use glib_ffi;
+use glib::GString;
```

Comment:
How did you get this all re-ordered, if not by rustfmt... but if rustfmt, why isn't everything else nicely formatted too then? :)

Reply:
I use rustfmt but stashed only this part 😉

Reply:
Maybe it's soon time to switch to it completely then ;) Anyway, fine with me to only take this change for now :)